lib/commonmark.js

```diff
@@ -86,7 +86,7 @@ function renderHtml(markdownString, options, cb) {
 }
 
 function parse(markdownString, cb) {
-  return cb(null, commonmarkRenderer.parse(markdownString));
+  return cb(null, commonmarkRenderer.parse(markdownString, {}));
```

Comment:
Ok for the first - I do not understand the second statement.
It *is* indeed calling `parse` directly - with the default apib options.